src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClient.cs

```diff
@@ -1,16 +1,16 @@
 ﻿namespace Amazon.SQS.ExtendedClient
 {
+    using Model;

```

Comment:
Please rearrange using as the were.
`System*` ones should come first.

Reply:
I think it's resharper that orders the imports alphabetically on save. I'll see if I can disable this.